docs/version.py

```diff
@@ -1,32 +1,32 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
 
+import git
 import sys
 from typing import List
 from packaging.version import parse, Version, InvalidVersion
-import requests
 import argparse
 
 
-def _get_releases(repo: str, organization: str = 'scipp') -> List[Version]:
+def _get_releases() -> List[Version]:
     """Return reversed sorted list of release tag names."""
-    max_tries = 3
-    ok = False
-    for n in range(max_tries):
-        r = requests.get(f'https://api.github.com/repos/{organization}/{repo}/releases')
-        ok = r.status_code == 200
-        if ok:
-            break
-    if not ok:
-        return []
-    data = r.json()
-    return sorted([parse(e['tag_name']) for e in data if not e['draft']], reverse=True)
+    tags = git.Repo('..').tags
+    versions = []
+    for t in tags:
+        try:
+            print(parse(t.name))
+            versions.append(parse(t.name))
+        except InvalidVersion:
+            pass
+    print(versions)
+    return sorted(versions, reverse=True)
 
 
 class VersionInfo:
 
-    def __init__(self, repo: str, organization: str = 'scipp'):
-        self._releases = _get_releases(repo=repo, organization=organization)
+    def __init__(self):
+        self._releases = _get_releases()
+        print('self._releases', self._releases)
```

Comment:
Did you leave the prints in here on purpose?

Reply:
no, just wanted to make sure they contained the right thing before removing them